package.json

```diff
@@ -51,8 +51,8 @@
     "sentry:publish": "node ./development/sentry-publish.js",
     "lint": "yarn lint:prettier && yarn lint:eslint && yarn lint:tsc && yarn lint:styles",
     "lint:fix": "yarn lint:prettier:fix && yarn lint:eslint:fix && yarn lint:styles:fix",
-    "lint:prettier": "prettier --check -- **/*.json",
-    "lint:prettier:fix": "prettier --write -- **/*.json",
+    "lint:prettier": "prettier --check -- '**/*.json'",
```

Comment:
This seems like it should be reverted?

Reply:
without the quotes, it causes `E2BIG error` when running `lint:prettier`